### PR TITLE
Legacy property set fixes

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
Legacy Property set fixes highlighted during deliverable import. I am doing this so we have the update noted in IFC Specification and also so they are not lost when the issue is closed.

Fixes Legacy items of #326

### CODE REVIEW

**Comment by AlexBrad1eyCT** [comment]
@SergejMuhic just a quick error check would be appreciated so we can get these merged and passed on...

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceCoveringRequirements/DocPropertySet.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceCoveringRequirements" UniqueId="7e5e0c89-424f-46bf-a2bb-cd0cf72e03c4" ApplicableType="IfcSpace" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceCoveringRequirements" UniqueId="7e5e0c89-424f-46bf-a2bb-cd0cf72e03c4" ApplicableType="IfcSpace,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceCoveringRequirements/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceFireSafetyRequirements/DocPropertySet.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceFireSafetyRequirements" UniqueId="0g3oO0qWaHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceFireSafetyRequirements" UniqueId="0g3oO0qWaHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceFireSafetyRequirements/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceLightingRequirements/DocPropertySet.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceLightingRequirements" UniqueId="2ltw40qWeHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceLightingRequirements" UniqueId="2ltw40qWeHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceLightingRequirements/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceOccupancyRequirements/DocPropertySet.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceOccupancyRequirements" UniqueId="32MTg0qWeHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceOccupancyRequirements" UniqueId="32MTg0qWeHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceOccupancyRequirements/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceParking/DocPropertySet.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceParking" UniqueId="3oeNa0qWeHuO00025QrE$V" ApplicableType="IfcSpace/PARKING" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceParking" UniqueId="3oeNa0qWeHuO00025QrE$V" ApplicableType="IfcSpace/PARKING,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceParking/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceThermalRequirements/DocPropertySet.xml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceThermalRequirements" UniqueId="0yroe0qWmHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpaceThermalRequirements" UniqueId="0yroe0qWmHuO00025QrE$V" ApplicableType="IfcSpace, IfcSpatialZone, IfcZone,IfcSpaceType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/PropertySets/Pset_SpaceThermalRequirements/DocPropertySet.xml]: Missing a space.

File: IFC4x3/Properties/r/Reference_09uhUrGP96Dg7LCTKoeQZI/DocProperty.xml [hunk 7]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_09uhUrGP96Dg7LCTKoeQZI" Name="Reference" UniqueId="09e2b7b5-4192-4636-a1d5-31d532a1a8d2" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_09uhUrGP96Dg7LCTKoeQZI" Name="Reference" UniqueId="09e2b7b5-4192-4636-a1d5-31d532a1a8d2" PrimaryDataType="IfcIdentifier" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/Reference_09uhUrGP96Dg7LCTKoeQZI/DocProperty.xml:2]: Did you check of all of these were `IfcIdentifier` before? This one was `IfcLabel`.

File: IFC4x3/Properties/r/Reference_1mSb87I6b0egBV57bTxz1d/DocProperty.xml [hunk 8]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_1mSb87I6b0egBV57bTxz1d" Name="Reference" UniqueId="70725207-4869-40a2-a2df-14795defd067" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_1mSb87I6b0egBV57bTxz1d" Name="Reference" UniqueId="70725207-4869-40a2-a2df-14795defd067" PrimaryDataType="IfcIdentifier" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/Reference_1mSb87I6b0egBV57bTxz1d/DocProperty.xml:2]: Was `IfcLabel` also.

File: IFC4x3/Properties/r/Reference_2KCLcfF8r35gUW6qE2nkg0/DocProperty.xml [hunk 9]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_2KCLcfF8r35gUW6qE2nkg0" Name="Reference" UniqueId="943159a9-3c8d-4316-a7a0-1b4382c6ea80" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_2KCLcfF8r35gUW6qE2nkg0" Name="Reference" UniqueId="943159a9-3c8d-4316-a7a0-1b4382c6ea80" PrimaryDataType="IfcIdentifier" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/Reference_2KCLcfF8r35gUW6qE2nkg0/DocProperty.xml:2]: `IfcLabel`

File: IFC4x3/Properties/r/Reference_36T0J90GnF1AFJynwsfDWJ/DocProperty.xml [hunk 10]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_36T0J90GnF1AFJynwsfDWJ" Name="Reference" UniqueId="c67404c9-010c-4f04-a3d3-f31eb6a4d813" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Reference_36T0J90GnF1AFJynwsfDWJ" Name="Reference" UniqueId="c67404c9-010c-4f04-a3d3-f31eb6a4d813" PrimaryDataType="IfcIdentifier" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/r/Reference_36T0J90GnF1AFJynwsfDWJ/DocProperty.xml:2]: `IfcLabel`

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Some changes needed in my view.

File: IFC4x3/Properties/r/Reference_09uhUrGP96Dg7LCTKoeQZI/DocProperty.xml (hunk 7 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/r/Reference_09uhUrGP96Dg7LCTKoeQZI/DocProperty.xml:2]: IfcIdentifier in leading edge docs from BSI online, it is deprecated anyway so no issue

File: IFC4x3/Properties/r/Reference_1mSb87I6b0egBV57bTxz1d/DocProperty.xml (hunk 8 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Properties/r/Reference_1mSb87I6b0egBV57bTxz1d/DocProperty.xml:2]: definately all Identifier in leading edge docs

**Comment by AlexBrad1eyCT** [comment]
Decision on adding Types to list.

**Comment by AlexBrad1eyCT** [comment]
Type Applicability conventions applied

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**